WalletWasabi.Tests/Helpers/ServiceFactory.cs

```diff
@@ -51,8 +55,21 @@ public static class ServiceFactory
 			throw new InvalidOperationException($"Coin clones have been detected. Number of all coins:{sCoins.Length}, unique coins:{uniqueCoins}.");
 		}
 
+		IEnumerable<SmartCoin>? allowedInputs = null;
+		if (allowedInputsIndexes is not null)
+		{
+			allowedInputs = new List<SmartCoin>();
+			foreach (var i in allowedInputsIndexes)
+			{
+				allowedInputs = allowedInputs.Append(sCoins[i]);
+			}
+		}
+
 		var coinsView = new CoinsView(sCoins);
 		var mockTransactionStore = new AllTransactionStore(".", Network.Main);
+
+		feeRateFetcher ??= () => new FeeRate(2m);
+
 		return new TransactionFactory(Network.Main, keyManager, coinsView, mockTransactionStore, password, allowUnconfirmed);
```

Comment:
```suggestion
		return new TransactionFactory(Network.Main, keyManager, coinsView, mockTransactionStore, password, allowUnconfirmed, allowDoubleSpend);
```

If we want to send `allowDoubleSpend` as a parameter, then we should forward it to the `TransactionFactory`.
Otherwise it's just an unused parameter.

Reply:
All newly added parameters will be used [here ](https://github.com/zkSNACKs/WalletWasabi/pull/11709)in `TransactionFactoryParameter` .

Reply:
OK, I see now.